src/Column/DataColumnRenderer.php

```diff
@@ -58,14 +58,20 @@ public function __construct(
     public function renderColumn(ColumnInterface $column, Cell $cell, GlobalContext $context): Cell
     {
         $this->checkColumn($column);
-        return $cell->addAttributes($column->columnAttributes);
+        /** @var DataColumn $column This annotation is for IDE only */
+
+        return $cell
+            ->addClass($column->columnClass)
+            ->addAttributes($column->columnAttributes);
```

Comment:
Change order `addAttributes` and `addClass`, because `addAttributes` override `class` attribute. Also need test for such case.